interfaces/coppeliasim/robots/LBR4pCoppeliaSimRobotZMQ.m

```diff
@@ -0,0 +1,119 @@
+%    CoppeliaSim. For instance, if you drag-and-drop a second robot to the
+%    scene, CoppeliaSim will rename the first robot to "/LBR4p[0]" and name
+%    the second one "/LBR4p[1]". A third robot will be named "/LBR4p[2]",
+%    and so on.
+
+classdef LBR4pCoppeliaSimRobotZMQ < DQ_SerialCoppeliaSimRobotZMQ
+    methods
+        function obj = LBR4pCoppeliaSimRobotZMQ(robot_name, coppeliasim_interface)
+            % This method constructs an instance of a KUKA LBR4+ robot.
+            % Usage:
+            %     LBR4pCoppeliaSimRobotZMQ(robot_name, coppeliasim_interface)
+            %          robot_name: The name of the robot in the CoppeliaSim
+            %          scene.
+            %          coppeliasim_interface: The DQ_CoppeliaSimInterfaceZMQ
+            %          object connected to the CoppeliaSim scene.
+            %
+            % Example: 
+            %      cs = DQ_CoppeliaSimInterfaceZMQ();
+            %      cs.connect;
+            %      robot_coppeliasim = LBR4pCoppeliaSimRobotZMQ("/LBR4p", cs);
+            %     
+            %    Note that the name of the robot should be EXACTLY the same
+            %    as in CoppeliaSim. For instance, if you drag-and-drop a
+            %    second robot to the scene, CoppeliaSim will rename the
+            %    first robot to "/LBR4p[0]" and name the second one
+            %    "/LBR4p[1]". A third robot will be named "/LBR4p[2]", and
+            %    so on.
+
+            arguments 
+                robot_name (1,:) {mustBeText}
+                coppeliasim_interface (1,1) DQ_CoppeliaSimInterfaceZMQ
+            end
+
+            obj@DQ_SerialCoppeliaSimRobotZMQ(robot_name, ...
+                coppeliasim_interface);
+        end
+
+        function kin = kinematics(obj)
+            % This method constructs an instance of a DQ_SerialManipulatorDH.
+            % Usage:
+            %     kinematics();
+            %
+            % Example: 
+            %      cs = DQ_CoppeliaSimInterfaceZMQ();
+            %      cs.connect;
+            %      robot_coppeliasim = LBR4pCoppeliaSimRobotZMQ("/LBR4p", cs);
+            %      robot_kinematics = robot_coppeliasim.kinematics();
+
+            arguments 
+                obj (1,1) LBR4pCoppeliaSimRobotZMQ
+            end
+
+            LBR4p_DH_theta = [0, 0, 0, 0, 0, 0, 0];
+            LBR4p_DH_d = [0.200, 0, 0.4, 0, 0.39, 0, 0];
+            LBR4p_DH_a = [0, 0, 0, 0, 0, 0, 0];
+            LBR4p_DH_alpha = [pi/2, -pi/2, pi/2, -pi/2, pi/2, -pi/2, 0];
+            LBR4p_DH_type = double(repmat( ...
+                DQ_SerialManipulatorDH.JOINT_ROTATIONAL, 1, 7));
+            LBR4p_DH_matrix = [LBR4p_DH_theta;
+                               LBR4p_DH_d;
+                               LBR4p_DH_a;
+                               LBR4p_DH_alpha
+                               LBR4p_DH_type];
+            
+            kin = DQ_SerialManipulatorDH(LBR4p_DH_matrix, 'standard');
```

Comment:
Hi @ffasilva, the constructor `DQ_SerialManipulatorDH(DH_matrix, 'standard') is deprecated. Please use the modern constructor of the class.